src/controllers/playback/videoosd.js

```diff
@@ -1100,6 +1100,20 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
             }
 
             switch (e.key) {
+                case "Enter":
+                    showOsd();
+                    break;
+
+                case "Escape":
+                case "RCUBack": // WebOS back
+                case "XF86Back": // Tizen back
```

Comment:
In `keyboardnavigation`, I added `getKeyName`, which can be used here to map keys - this will also fix lack of `key` field in `KeyboardEvent` on old browsers (WebOS 2/3).

Reply:
Why did't you use it here already, is there a reason for it?

Reply:
Just not sure. Does it (import `keyboardnavigation` here) look good?

Reply:
For me it looks okay, @MrTimscampi is now working on re-ordering imports, maybe he have a say about that?

Reply:
No worries with imports on my end, it's only module definitions that were an issue :)

Reply:
No issues on my end, the changes to modules are only related to define() (One per file, have to respect relative path, extensions needed for css! etc)